lib/web_ui/lib/src/engine/canvaskit/rasterizer.dart

```diff
@@ -2,80 +2,54 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:meta/meta.dart';
 import 'package:ui/src/engine.dart';
 import 'package:ui/ui.dart' as ui;
 
 /// A class that can rasterize [LayerTree]s into a given [Surface].
 class Rasterizer {
+  Rasterizer(this.view);
```

Comment:
It looks like the only thing you need from `view` is the `sceneHost`. Does it make sense to pass only the `DomElement sceneHost` instead of the whole `EngineFlutterView`?

(ditto for `HtmlViewEmbedder`)

Reply:
Done